Felienne/hedy
# Adds button to create public profile when not created yet

**Description**
In this PR we add a button to the public profile container on the landing page if the user doesn't have a public profile yet. This button redirects to the profile page with the public profile section automatically open.

**Fixes**
This PR fixes #3042.

**How to test**
Navigate to the `/landing-page` with an account without a public profile. Verify that there is new button and that it redirects to the profile page with the public profile section expanded.
